### PR TITLE
net: buf: Fix debug level setting

### PR DESCRIPTION
**Pull request by jukkar**
Some of the net_buf related tests failed if they enabled
CONFIG_NET_BUF_LOG setting.

Signed-off-by: Jukka Rissanen <jukka.rissanen@linux.intel.com>

### CODE REVIEW

**Comment by pfalcon** [comment]
@jukkar, github keeps glitching, yeah. New tickets 404 until they "ripe up" for awhile. Closing/resubmitting imho doesn't make sense IMHO. YMMV.

(Can't see diff on this PR in the meantime, 404.)

**Comment by jukkar** [comment]
> github keeps glitching, yeah. New tickets 404 until they "ripe up" for awhile. Closing/resubmitting imho doesn't make sense IMHO. YMMV.

Yes, it seems so indeed. It was just that the first PR about this just vanished and I could not access it so I submitted a new one just to make sure it is there.

File: subsys/net/Kconfig [hunk 1]
```diff
@@ -33,9 +33,10 @@ config NET_BUF_LOG
 
 module = NET_BUF
 module-dep = LOG
+module-def = LOG_LEVEL_WRN
```
Comment (pfalcon) [inline subsys/net/Kconfig]: This is not strictly related to this patch, but overall. This "module-def" bothers me. Nobody knows that it's "default", everyone thinks it's "define".

Comment (pfalcon) [reply subsys/net/Kconfig]: Well, someone may look once, then forget and get confused again. I suggest that we change it and make self-descriptive (in a separate patch of course).

Comment (jukkar) [reply subsys/net/Kconfig]: I suggest this is done like this for now. If needed there could be a separate PR that changes all the invocations of `module-def` later.

**Comment by jukkar** [comment]
recheck

**Review by rveerama1 — APPROVED** [review]
LGTM
